### PR TITLE
fix(console): remove annoying horizontal scrollbar from code editor

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Remove this annoying horizontal scrollbar from code editor

image

## Testing

- [x] Tested locally and the scrollbar was gone

### CODE REVIEW

File: packages/console/src/components/CodeEditor/index.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ const CodeEditor = ({ className, language, isReadonly = false, value = '', onCha
             borderRadius: '0',
             wordBreak: 'break-all',
             fontFamily: "'Roboto Mono', monospace", // Override default font-family of <pre>
+            overflow: 'hidden',
```
Comment (gao-sun) [inline packages/console/src/components/CodeEditor/index.tsx:56]: What if the content is too long (many rows)?

Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:56]: Currently we don't show scrollbars inside code editor. If the content is too long, then we scroll on the outer container instead

Comment (simeng-li) [reply packages/console/src/components/CodeEditor/index.tsx:56]: how about, :hover: overflow-y auto

Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:56]: No... I don't think we need overflow on the `textarea` and `pre` elements. If we really want to scroll, we scroll on the outer `div` container

Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx]: Code updated. Decided to use `display: flex;` in CSS instead

**Review by gao-sun — APPROVED** [review]

**Comment by Alleged627** [comment]
Awesome thanks

**Pull request merged**